samples/mixed/consumer/src/main.rs

```diff
@@ -136,78 +136,101 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
 
     info!("The Consumer has started.");
 
+    let settings = load_settings();
```

Comment:
maybe change this to `let settings = consumer_config::load_settings();` so that it clear that the consumer is utilizing that version of load settings since the provider_config also has the same function and it can be easily confused and isnt immediately obvious in the code which is being used

Reply:
Changed.